### DISCUSSION TITLE
Problem installation

### DISCUSSION THREAD

**Discussion by mly-h** [monarc-project/MonarcAppFO#511] [Show and tell]
Hello, I tried the installation of Monarc on Ubuntu but when I try to access the interface I have a blank page. And when I specify the folder in the url (add_ip/monarc) it shows me an error. Do you have any idea what the problem is?
![image](https://github.com/monarc-project/MonarcAppFO/assets/133865281/12a6d8ef-7945-4499-b25e-abf821a1cdec)

**Comment by ruslanbaidan** [comment]
Hello,
There are a few possibilities to determine the exact issue:
- check apache error.log
- enable errors displaying in php.ini or on the the app, e.g. in `public/index.php`
```
ini_set('display_errors', 1);
error_reporting(E_ALL);
```
Usually this kind of problems could be related to incomplete installations (database settings, missing data/cache... directories or permissions problems) or apache virtual host configuration.

**Reply by mly-h** [reply]
Hello, thank you for the response. Indeed, it was a permission problem. I can now access the graphical interface, but a new problem has arisen: the list of existing models and languages to create a risk analysis is empty
![image](https://github.com/monarc-project/MonarcAppFO/assets/133865281/ac2aa95e-f4bf-47a1-ac80-ec3a710b5924)

**Reply by ruslanbaidan** [reply]
Hello,
Languages appear only after model selection. Models are taken from the moanrc_common database.
It seems the initial data were not populated, it's [here](https://github.com/monarc-project/MonarcAppFO/blob/master/INSTALL/INSTALL.ubuntu2004.md#initializes-the-database)

**Reply by mly-h** [reply]
Thank you for the feedback. I had already executed the commands. So, I deleted the 'monarc_common' database and re-ran the mentioned commands, but still, nothing appears in the model selection. I also created a new user, but it doesn't appear in the user management seeting. I also noticed that the yellow loading bar never reaches the end.
![image](https://github.com/monarc-project/MonarcAppFO/assets/133865281/103d4938-d20d-4dbd-b325-2b45356e5508)

**Reply by ruslanbaidan** [reply]
To be able to see the application errors, there is a browser Developer Tools console.
All the requests from frontend to backend are tracked on the Network tab.

Could you try to turn off php deprecation notices and warnings but updating your php.ini file, the following line:
`error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT & ~E_WARNING & ~E_NOTICE`
after the apache reload is required.
Maybe the missing models in the list could be related to that, if not you could connect to the database and query the models table in the `monarc_common` db.

**Reply by mly-h** [reply]
Hello, thank you very much for your help; everything is working well now.
